crafters/image/AlbumentationsCrafter/README.md

```diff
@@ -4,6 +4,40 @@ This crafter provides access to any of the transforms from the [Albumentations](
 
 > Note that Albumentation's transform are meant to be random, which is usually not what you want in indexing/search context. The crafter automatically adds `always_apply=True` to all transforms, but you should watch out for other sources of "randomness" (see AlbumentationsCrafter documentation for more detail)
 
+## Usage
+
+Users can use Pod images in several ways:
+
+1. Run with Docker (`docker run`)
+   ```bash
+    docker run jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-in 55555 --port-out 55556
```

Comment:
When running directly with docker, one needs to either bind the container port or to use `--network host`.
```suggestion
    docker run --rm -p 55555:55555 -p 55556:55556 jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-in 55555 --port-out 55556
```

Reply:
yes sure, added the suggestion